src/sql/util.py

```diff
@@ -250,56 +250,89 @@ def parse_toml_error(e, file_path):
         return e
 
 
-def get_user_configs(file_path):
+def get_user_configs(primary_path, alternate_path):
     """
     Returns saved configuration settings in a toml file from given file_path
 
     Parameters
     ----------
-    file_path : str
+    primary_path : Path
+        file path to a toml file
+    alternate_path : Path
```

Comment:
it's not clear what's the difference b/w `primary_path` and `alternate_path`. Both mention `file path to a toml file`

Reply:
Fixed [here](https://github.com/ploomber/jupysql/pull/935/files#diff-653c15e6161c97f356be04e072838ed15014af275c06019a7692ab727fa44b97R372-R375)